examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -7451,21 +7523,57 @@ endpoint 1 {
   }
 }
 endpoint 2 {
+  device type ma_powersource = 17, version 1;
   device type ma_onofflight = 256, version 1;
 
 
+  server cluster Identify {
```

Comment:
This is ZAP enforcing spec requirements (onofflight must have Identify and scenes).

That said, I expect this Identify impl is broken, since the app does not actually set it up.  Maybe file a followup on that?

Reply:
I was waffling back and forth about just taking the changes for the device type without all the other changes from zap. My concern was that we're already out of sync between the gui and the saved files, and I didn't want to exacerbate the problem. However, it seems like perhaps this could cause issues. Thoughts?

Reply:
I think this change to the .zap is fine: we should not have our example apps violating the spec (though all-clusters is "Special" that way already....).  Just file a followup to make the impl part work, please.